src/main/java/com/wiley/autotest/selenium/elements/upgrade/v3/conditions/window/WindowByPartialUrl.java

```diff
@@ -0,0 +1,19 @@
+package com.wiley.autotest.selenium.elements.upgrade.v3.conditions.window;
+
+import com.wiley.autotest.ExpectedConditions2;
+import org.openqa.selenium.WebDriver;
+
+import java.util.function.Function;
+
+public class WindowByPartialUrl extends WindowFinder {
+
+    public WindowByPartialUrl(String locatedBy) {
+        super(locatedBy);
+    }
+
+    @Override
+    public Function<WebDriver, String> findAndSwitch() {
+        waitForChrome();
```

Comment:
is this still actual? I would suggest removing this, you can add unit tests to make sure it works without it as well.